### PR TITLE
csv export button

### PR DESCRIPTION
**Pull request by anyajain5**
# Pull Request

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

### Code Review Preparation
- [x] I have performed a self-review and reviewed the changed files
- [x] My code follows the project's style guidelines and passes linting
- [ ] I have added comments in hard-to-understand areas and updated documentation

### Testing (Ignore for the first PR)
- [ ] I have added relevant tests and they pass locally
- [ ] I have tested the changes manually, including edge cases

### Code Quality & Security
- [x] I have formatted my code and removed debugging/unused code
- [x] No TypeScript errors or linting issues
- [x] No sensitive information committed (API keys, passwords, etc.)
- [x] User inputs are validated and sanitized

### Dependencies & Deployment
- [ ] Dependencies are documented and package-lock.json is updated if needed
- [x] Changes work with Firebase emulator and environment variables are documentedRetry

## Summary by CodeRabbit

* **New Features**
  * Export control added to the header allowing download of All Accounts and Previous Donors as CSV.

* **Refactor**
  * Header reworked into a two-column layout with improved spacing and alignment.
  * Top navigation reordered and "Past Donors" renamed to "Previous Donors" for clearer terminology.

### CODE REVIEW

File: app/user-management/all-accounts/page.tsx [hunk 1]
```diff
@@ -22,9 +23,34 @@ export default function AllAccountsPage() {
...
+        const csv = [headers, ...rows].map(row => row.join(",")).join("\n");
+
+        const blob = new Blob([csv], { type: "text/csv" });
+        const url = URL.createObjectURL(blob);
+        const a = document.createElement("a");
+        a.href = url;
+        a.download = "users.csv";
+        a.click();
+        URL.revokeObjectURL(url);
+    }
```
[the remark this answers is not in the document]
Comment (constrictingsnake) [reply app/user-management/all-accounts/page.tsx]: the locale date string isnt too important but the csv corruption thing is something to fix

File: app/user-management/past-donors/page.tsx [hunk 2]
```diff
@@ -5,15 +5,47 @@ import { DonorsTable } from "@/components/user-management/DonorsTable";
...
+        const headers = ["First Name", "Last Name", "Email", "Phone Number", "Street Address", "City", "State", "Zip Code"];
+        const rows = donors.map(donor => [
+            donor.firstName,
+            donor.lastName,
+            donor.email,
+            donor.phoneNumber,
+            donor.address.streetAddress,
+            donor.address.city,
+            donor.address.state,
+            donor.address.zipCode,
```
Comment (constrictingsnake) [inline app/user-management/past-donors/page.tsx]: with the current implementation, the zip code gets cut off. (ex: 06112 becomes 6112)

**Review by constrictingsnake — CHANGES_REQUESTED** [review]
